x-pack/plugins/alerting/server/alert/alert.ts

```diff
@@ -46,18 +54,24 @@ export class Alert<
   private state: State;
   private context: Context;
   private readonly id: string;
+  private readonly uuid: string;
```

Comment:
Defensively storing the `uuid` in the alert, as we occaisonally mutate `meta`, so to make sure we don't get an unexpected uuid from a meta mutation, store in the alert, and copy it into meta when the alert is created.